package.json

```diff
@@ -9,7 +9,14 @@
     "_metalsmith": "bin/_metalsmith"
   },
   "scripts": {
-    "test": "make test"
+    "pretest": "npm install",
+    "test": "mocha $MOCHA_OPTS",
+    "pretest-debug": "npm install",
+    "test-debug": "mocha debug",
+    "precoverage": "npm install",
+    "coverage": "istanbul cover ./node_modules/.bin/_mocha",
```

Comment:
Maybe the coverage part should actually go into the [`deploy` section] with a restriction to one node version.

[`deploy` section]: https://docs.travis-ci.com/user/deployment/